src/script/interpreter.cpp

```diff
@@ -1189,7 +1358,14 @@ uint256 GetPrevoutHash(const T& txTo)
     for (const auto& txin : txTo.vin) {
         ss << txin.prevout;
     }
-    return ss.GetHash();
+    return ss.GetSHA256();
+}
+
+uint256 HashAgain(const uint256& hash)
+{
+    uint256 result;
+    CSHA256().Write(hash.begin(), 32).Finalize(result.begin());
+    return result;
 }
```

Comment:
Slightly confusing name... `HashAgainSHA256`?

I understand why it exists, and I can't think of a much better name... but having something in the name to denote we're hashing with SHA256 might make it a tiny bit more readable.

Reply:
Changed into `SHA256Uint256`, and moved to hash.{h,cpp}.